src/frontend/parser.mly

```diff
@@ -810,3 +810,13 @@ top_vardecl_or_statement:
     { grammar_logger "top_vardecl_or_statement_statement" ; [s] }
   | v=top_var_decl
     { grammar_logger "top_vardecl_or_statement_top_vardecl" ; v }
+
+arg_list_trailing_comma(sep, X):
```

Comment:
Can we make this %inline?

Also, if we want to name it `arg_list_trailing_comma` I don't think we need to take in sep, X as arguments, we can hard-code them as COMMA and arg_decl.

If we want to use this anywhere else (I can't think of any such place at the moment) we should keep it as a higher-order rule and rename it to something like `seperated_list_trailing`